executor/metatron/metatron.go

```diff
@@ -170,19 +171,14 @@ func createPassportDir(taskID string) error {
 	return os.MkdirAll(getMetatronOutputPath(taskID), os.FileMode(0700))
 }
 
-// RemovePassports removes a task's Metatron credential directory on the host
-func RemovePassports(taskID string) error {
```

Comment:
I notice we removed the call to this function back in May (https://github.com/Netflix/titus-executor/commit/885ba4bfb9a8111b2a34c263c61e50baaeb80a3c#diff-957e82dddf339c98741109a28268e372). Does that mean we are leaving these creds laying around the host? I realize this is unrelated to this specific PR.

Reply:
Yeah. We are. It'd be nice to move to a model where the agent never has to write / read metatron creds from disk, and instead the metatron agent wrote a tarball to stdout.

Reply:
Yes, I think you may be right.